### ISSUE TITLE
Feature Cycle (crash), with Magical Forest + Tectonic

### ISSUE THREAD

**Issue by alcatrazEscapee** [ApolloDatapacks/tectonic#14]
This was reported to me (https://github.com/alcatrazEscapee/cyanide/issues/15), but Cyanide just caught the feature cycle and displayed the issue.

- Crash Log: [crash-2023-01-29_10.11.00-client.txt](https://github.com/DenisMasterHerobrine/MagicalForest/files/10530090/crash-2023-01-29_10.11.00-client.txt)
- Minecraft 1.18.2
- Magical Forest `MagicalForest-forge-1.18.2-2.0.3.jar`
- Tectonic `tectonic-mod-mc1.18.2-v1.1.3.jar`

The feature cycle:

```
Feature 'tectonic:common/seagrass_land'
  must be before 'tectonic:common/grass/patch_grass_dense' (defined in 'tectonic:old_growth_evergreen_forest' at index 5, 6 and 9 others)
  must be before 'minecraft:brown_mushroom_taiga' (defined in 'minecraft:taiga' at index 4, 5)
  must be before 'minecraft:red_mushroom_taiga' (defined in 'byg:frosted_taiga' at index 4, 5 and 10 others)
  must be before 'minecraft:patch_grass_plain' (defined in 'magicalforest:magical_forest' at index 7, 8)
  must be before 'minecraft:flower_meadow' (defined in 'magicalforest:magical_forest' at index 8, 9)
  must be before 'tectonic:common/seagrass_land' (defined in 'minecraft:meadow' at index 3, 4 and 1 others)
```

* Note that the `byg:frosted_tagia` is not necessary, as this ordering is also defined by Tectonic in it's override of `minecraft:tagia`

**Basically**

- Tectonic defines `tectonic:common/seagrass_land` **before** `minecraft:brown_mushroom_tagia`, and **after** `minecraft:flower_meadow`, in two separate places.
- Magical forest defines these vanilla features in the opposite order.
- Vanilla, as far as I can see, does not a priori define an ordering between these.

Also reported as https://github.com/DenisMasterHerobrine/MagicalForest/issues/10

---

As a side note, would you consider linking either your source/issues page from your curseforge page, or linking the modrinth page (which has both) from curseforge? As it stands I found the curseforge page, then had to find a discord, find a announcement, find a modrinth page, then find a source + issues location which was rather confusing.

**Comment by ApolloDatapacks** [comment]
The feature order cycle has been fixed (`tectonic:common/seagrass_land` was moved before both of them). Curseforge now has source/issues links as well.
